pkg/controller/kubelet-config/kubelet_config_controller.go

```diff
@@ -475,6 +476,18 @@ func (ctrl *Controller) syncKubeletConfig(key string) error {
 		klog.V(4).Infof("Finished syncing kubeletconfig %q (%v)", key, time.Since(startTime))
 	}()
 
+	if err := wait.PollUntilContextTimeout(context.TODO(), ctrlcommon.ControllerConfigRolloutInterval, ctrlcommon.ControllerConfigTimeout, false, func(_ context.Context) (bool, error) {
```

Comment:
suggestion: Hoist this wait code into a separate function for easier reuse since it looks like it is being used in multiple places.

```golang
func waitForControllerConfigCreation() error {
	return wait.PollUntilContextTimeout(context.TODO(), ctrlcommon.ControllerConfigRolloutInterval, ctrlcommon.ControllerConfigTimeout, false, func(_ context.Context) (bool, error) {
		// ...
	}
}
```